### PR TITLE
Passport validation - check with snapshot block

### PR DESCRIPTION
**Pull request by ChaituVR**

### CODE REVIEW

File: src/validations/passport/index.ts [hunk 1]
```diff
@@ -9,19 +14,30 @@ export default class extends Validation {
...
 
     let weight = 0;
     this.params.stamps.forEach((stamp: any) => {
-      const found = verifiedStamps.find((s: any) => s.provider === stamp.id);
-      if (found?.verified) weight += stamp.weight;
+      const verifiedStamp = verifiedStamps.find(
+        (s: any) => s.provider === stamp.id
+      );
+
+      // check that the credential is still valid (created after snapshot block and not expired)
```
Comment (bonustrack) [inline src/validations/passport/index.ts:35]: What you mean "created after snapshot block", we should check it user were attested before or in the block number, not after

Comment (ChaituVR) [reply src/validations/passport/index.ts]: oops , typo in comment, fixed it 👍

**Comment by bonustrack** [comment]
So this does not address the issue with multiple addresses having same attestation right?

**Comment by ChaituVR** [comment]
Yep this is just to enforce snapshot block

**Review by bonustrack — APPROVED** [review]
utACK

**Pull request merged**